include/seqan3/utility/range/to.hpp

```diff
@@ -0,0 +1,200 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2022, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2022, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include <seqan3/std/ranges>
+
+#include <seqan3/core/platform.hpp>
+#include <seqan3/core/range/detail/adaptor_from_functor.hpp>
+#include <seqan3/utility/concept.hpp>
+
+#include <range/v3/range/conversion.hpp>
+
+namespace seqan3::detail
+{
+
+/**
+ * Function to convert a std::ranges::range to a fully defined container
+ */
+template <typename container_t>
+struct to_fn
+{
+private:
+    /*!\brief Copy some range into a container
+     *
+     * \tparam rng_t       type of the range
+     * \tparam container_t type of the target container
+     */
+    template <std::ranges::range rng_t>
+    auto impl(rng_t && rng, container_t & container) const
+    {
+        using std::ranges::copy;
+        copy(rng, back_inserter(container));
```

Comment:
```suggestion
        std::ranges::copy(rng, std::back_inserter(container));
```

And elsewhere, or do you have a specific reason to rely on ADL?